### DISCUSSION TITLE
Prisma migrate with unsupported field

### DISCUSSION THREAD

**Discussion by shulmaneyal** [prisma/prisma#11495] [Q&A] [answered]
I have a model with an unsupported field, when using prisma migrate, prisma produces the correct sql statements for adding this field but then every time I run a new migration without changing the model it adds an alter column statement for the field, is there a way to avoid it?

Sample model:
```
model Folder {
  id         String                             @id @default(uuid()) @db.NVarChar(36)
  parsedPath String
  path       Unsupported("[sys].[hierarchyid]")
}
```

New migration alter statement:
```
ALTER TABLE [dbo].[Folder] ALTER COLUMN [path] [sys].[hierarchyid] NOT NULL;
```

**Comment by janpio** [comment, marked as answer]
Yes, report a bug with all the information you can provide so we can take a loot at it and fix it - this looks broken.

**Reply by shulmaneyal** [reply]
Opened issue [#11499](https://github.com/prisma/prisma/issues/11499)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
